railties/lib/rails/generators/erb/scaffold/templates/index.html.erb.tt

```diff
@@ -16,7 +16,7 @@
     <%% @<%= plural_table_name %>.each do |<%= singular_table_name %>| %>
       <tr>
 <% attributes.reject(&:password_digest?).each do |attribute| -%>
-        <td><%%= <%= singular_table_name %>.<%= attribute.name %> %></td>
+        <td><%%= <%= singular_table_name %>.<%= attribute.name %><% if attribute.reference? %>&.id<% end %> %></td>
```

Comment:
This will generate something like the following:

```erb
<td><%= comment.post&.id %></td>
```

We don't need to load the associated record to get its ID, we can use the value from the column directly:

```erb
<td><%= comment.post_id %></td>
```

The `column_name` method appears to exist for this purpose:
https://github.com/rails/rails/blob/f7fd680a6de195d78286332c6021afe068ad6bd6/railties/lib/rails/generators/generated_attribute.rb#L119-L121

Reply:
Good point! I guess you can use `attribute.column_name` instead of `attribute.name` then?

Reply:
Perfect. Much cleaner. Updated, rebased, and pushed.